### DISCUSSION TITLE
Getting live position data of a Solid Node in Webots using Python

### DISCUSSION THREAD

**Discussion by niklas030** [cyberbotics/webots#6304] [General]
Hello everyone,

I've been working on a project using the Webots simulation software and I've come across a problem that I haven't been able to solve. I'm looking to get live position data of a Solid node within the Webots environment. I want to continuously monitor this data so that my robot's controller can make decisions based on it.

I've been trying to find resources or examples online that demonstrate how to achieve this but so far, I've had no luck. I understand that Webots provides a Supervisor node that has the ability to access and control different aspects of the simulation, but I'm unsure how to use it to get real-time position data of a specific Solid node.

Does anyone have experience with this or can point me to the right resources? I'm specifically using Python for my robot's controller.

Any help would be greatly appreciated. Looking forward to your suggestions and discussions!

**Comment by omichel** [comment]
Please post technical questions to https://robotics.stackexchange.com, we will answer you there.
